prep.sh

```diff
@@ -8,10 +8,22 @@ cd "$(dirname "$0")"
 # releases, ideally implemented in the Makefile. Then both this repository and
 # jenkins-infra/release should be refactored to consume the new functionality.
 
-if [[ ! -f $WAR ]]; then
+if [[ ! -f "$WAR" ]]; then
+	# jv utilizes the JENKINS_VERSION environment variable which can be the line (latest/weekly/lts/stable) or an exact version
 	jv download
 fi
 
+if [[ ! -f "${WAR}.asc" ]]; then
+	# jv utilizes the JENKINS_VERSION environment variable which can be the line (latest/weekly/lts/stable) or an exact version
+	jenkinsVersion="$(jv get)"
+
+	# Download signature from Artifactory (signed by Maven during the release process)
+	# TODO: switch to get.jenkins.io once https://github.com/jenkins-infra/helpdesk/issues/4055 is finished
+	warSignatureUrl="https://repo.jenkins-ci.org/releases/org/jenkins-ci/main/jenkins-war/${jenkinsVersion}/jenkins-war-${jenkinsVersion}.war"
```

Comment:
Doesn't this need to end with `.asc` so that it downloads the signature instead of downloading the entire war file?
```suggestion
	warSignatureUrl="https://repo.jenkins-ci.org/releases/org/jenkins-ci/main/jenkins-war/${jenkinsVersion}/jenkins-war-${jenkinsVersion}.war.asc"
```

Reply:
Good catch. Looks like you fixed it with https://github.com/jenkinsci/packaging/pull/462/commits/425c92b3f6c4cd34ceef714169acf0f7a5f99fad, thanks!